Pipfile

```diff
@@ -8,10 +8,15 @@ arxiv-base = "*"
 flask = "*"
 requests = "*"
 arxiv-auth = "==0.1.0rc6"
+"boto3" = "*"
+moto = "==1.3.4"
```

Comment:
I pinned this, because ``1.3.5`` has an internal  dependency conflict: https://github.com/spulec/moto/issues/1813